### ISSUE TITLE
Memory leak with client:allFlags

### ISSUE THREAD

**Issue by heydemo** [launchdarkly/lua-server-sdk#19]
I've noticed calling client:allFlags leads to greater memory usage that is never released, causing our openresty instances to eventually run out of memory and crash. This is reproducible with the following script

```
local ld = require("launchdarkly_server_sdk")

local config = {
    key = SDK_KEY
}

local client = ld.clientInit(config, 1000)

local user = ld.makeUser({
    key = "abc"
})

local flags
while (true) do
  flags = client:allFlags(user)
end
```

The process just consumes more memory indefinitely.

**Comment by bwoskow-ld** [comment]
Hey @heydemo,

Thanks for reporting this. Can you identify for me your version of the Lua SDK as well as the version of the underlying C SDK? The reason for needing two SDK versions is because our Lua SDK it just a wrapper around our C SDK. Also, are you connected to a persistent feature store (Redis)?

**Comment by heydemo** [comment]
Thanks for the quick response!

Using c-server-sdk 2.21 and lua-server-sdk 1.0.1

Not using Redis.

**Comment by bwoskow-ld** [comment]
Thanks for the info @heydemo. We haven't (yet) detected such a memory leak in our integration testing suite where we run the SDKs in long-running applications, but we'll take a look. Thank you for the script to reproduce the issue.

It is worth noting that there are newer versions of both `c-server-sdk` and `lua-server-sdk` however I don't see any release notes pertaining to memory leak fixes after your currently running versions.

_Filed internally as 129668._

**Comment by cwaldren-ld** [comment]
Hi @heydemo, this has been resolved in [1.2.0](https://github.com/launchdarkly/lua-server-sdk/releases/tag/1.2.0). Thank you for your patience.
